pkgs/os-specific/windows/msvcSdk/update.nu

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env nix-shell
 #!nix-shell -i nu -p nushell xwin
+#!nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/f2edb71fbfb1dae0546873c9798551a71bc3e78b.tar.gz
```

Comment:
`git grep 'nix-shell -I nixpkgs='` make me think that we really ought to be using a relative path to the current Nixpkgs for this?

Reply:
Oh I forgot to remove that, it was just so I could draft this update before xwin was updated